### PR TITLE
Do not merge: non-human animal disease branch 

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
- add new (temp) relationship: cross-species analog
- add new term human disease
- add logical defs to:
    - 'bovine respiratory disease complex'
    - 'cattle disease'
    - cat disease 
    - 'leukemia, feline'
- add seeAlso to uPheno ticket 

address #5291

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -421008,8 +421012,9 @@ synonym: "Feline leukemias" RELATED [MESH:D016582]
 synonym: "leukemias, Feline" RELATED [MESH:D016582]
 xref: MESH:D016582 {source="MONDO:equivalentTo"}
 xref: UMLS:C0085164 {source="MONDO:equivalentTo"}
-is_a: MONDO:0005059 {source="MESH:D016582"} ! leukemia
 is_a: MONDO:0024912 {source="MESH:D016582"} ! cat disease
+intersection_of: MONDO:0024912 ! cat disease
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:421016]: I dont think you need that. It should be:

"disease" (MONDO:1) and
in taxon some cat and
has analog some leukemia

This is at least how I understood this?

Comment (matentzn) [reply src/ontology/mondo-edit.obo:421016]: this makes the patterns much easier to manage.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -98130,8 +98131,9 @@ xref: DOID:3732 {source="EFO:0007180", source="MONDO:obsolete"}
 xref: EFO:0007180 {source="MONDO:equivalentTo"}
 xref: MESH:D048090 {source="EFO:0007180", source="MONDO:equivalentTo"}
 xref: UMLS:C1449809 {source="MONDO:equivalentTo"}
-is_a: MONDO:0005087 {source="EFO:0007180", source="MESH:D048090/inferred"} ! respiratory system disorder
 is_a: MONDO:0024913 {source="MESH:D048090"} ! cattle disease
+intersection_of: MONDO:0024913 ! cattle disease
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:98135]: again see below. Just MONDO:1

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:421016]: ah, got it. I wasn't sure, this makes sense, thanks.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:98135]:
```suggestion
intersection_of: MONDO:0000001 ! disease or disorder
```

**Comment by nicolevasilevsky** [comment]
what is the right way to add the parentheses?

image


image

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:98135]: fixed

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:421016]: fixed

**Comment by sabrinatoro** [comment]
This PR can be closed as the work was done in another one.

**Pull request closed without merging**